src/replication.c

```diff
@@ -2513,6 +3129,16 @@ int replicaTryPartialResynchronization(connection *conn, int read_reply) {
         return PSYNC_TRY_LATER;
     }
 
+    if (!strncmp(reply, "+DUALCONNECTIONSYNC", 15)) {
```

Comment:
This string isn't 15 chars long.

I noticed you changed this string without changing the number. Hard-coded lengths aren't very maintenance friendly. :)

```suggestion
    if (!strncmp(reply, "+DUALCONNECTIONSYNC", strlen("+DUALCONNECTIONSYNC")) {
```